src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -21,7 +21,8 @@ import Link from 'app/components/links/link';
 import NavTabs from 'app/components/navTabs';
 import Placeholder from 'app/components/placeholder';
 import SeenByList from 'app/components/seenByList';
-import {IconEdit, IconTelescope, IconWarning} from 'app/icons';
+import {IconTelescope, IconWarning, IconLink} from 'app/icons';
+import {SectionHeading} from 'app/components/charts/styles';
```

Comment:
We already had something like SideHeader this that we use in events / discover that we can reuse here